idl/github.com/uber/cadence/history.thrift

```diff
@@ -815,6 +819,19 @@ service HistoryService {
       5: shared.ClientVersionNotSupportedError clientVersionNotSupportedError,
     )
 
+  /**
+  * GetDLQReplicationMessages return replication messages based on dlq info
+  **/
+  replicator.GetDLQReplicationMessagesResponse GetDLQReplicationMessages(1: replicator.GetDLQReplicationMessagesRequest request)
+    throws (
+      1: shared.BadRequestError badRequestError,
+      2: shared.InternalServiceError internalServiceError,
+      3: shared.LimitExceededError limitExceededError,
+      4: shared.ServiceBusyError serviceBusyError,
+      5: shared.ClientVersionNotSupportedError clientVersionNotSupportedError,
+      6: shared.EntityNotExistsError entityNotExistError,
```

Comment:
same as above
when will EntityNotExistsError be returned

Reply:
When the workflow passed the retention period and mutable state is not found